packages/host/app/services/loader-service.ts

```diff
@@ -98,11 +98,30 @@ export default class LoaderService extends Service {
     } else {
       body = await request.text();
     }
-    return await this.loader.fetch(request.url, {
+    let response = await this.loader.fetch(request.url, {
       method: request.method,
       headers: new Headers(request.headers),
       body,
     });
+
+    // We will get a 401 from the server in two cases. The first one is when the request is not allowed,
```

Comment:
Is this comment correct? Won't we get a 403 when request is not allowed? And 401 when token is invalid/expired/payload outdated?